logging/src/main/java/com/virtuslab/logger/PrefixedLambdaLogger.java

```diff
@@ -0,0 +1,77 @@
+package com.virtuslab.logger;
+
+import java.util.function.Supplier;
+
+import kr.pe.kwonnam.slf4jlambda.LambdaLogger;
+
+public class PrefixedLambdaLogger implements IPrefixedLambdaLogger {
+  private final LambdaLogger logger;
+  private final String className;
+
+  PrefixedLambdaLogger(LambdaLogger logger) {
+    this.logger = logger;
+    // We are sure that at the moment when this constructor is called we have at least 4 elements in stacktrace array:
+    // #0: java.lang.Thread#getStackTrace
+    // #1: com.virtuslab.logger.MacheteLogger#<init>
+    // #2: com.virtuslab.logger.MacheteLoggerFactory#getLogger
+    // #3: method from MacheteLoggerFactory#getLogger was called
+    @SuppressWarnings("upperbound")
```

Comment:
Likely you need `upperbound:<something>` for this suppression to have any effect

Reply:
no, it has effect anyway